plugins/register-component/src/components/RegisterComponentPage/RegisterComponentPage.tsx

```diff
@@ -85,7 +85,8 @@ export const RegisterComponentPage = ({
     const { componentLocation: target } = formData;
     try {
       const typeMapping = [
-        { url: /https:\/\/gitlab\.com\/.*/, type: 'gitlab' },
+        /** Make support self-hosted gitlab */
+        { url: /gitlab*/g, type: 'gitlab' },
```

Comment:
Hey @hooloovooo 👋 
No problem with delay ☺️

If #2426 resolve this problem we can close PR
So i'm not sure what #2426 resolse regexp mathing self-hosted gitlab

Here i change regexp to global find `gitlab` entry in domain. Also this won't work for situations, where i'll host gitlab on domain, ex: https://my-git.org.com/

Reply:
#2426 Only adds this to the scaffolder. We could base the techdocs support for gitlab off of that, but i think this implementation is a lot more in line with what we already have in techdocs. It also adds some things we've discussed earlier, such as creating our own default-branch utility.